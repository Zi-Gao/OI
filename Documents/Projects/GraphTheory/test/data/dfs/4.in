105 830
28 64 21605
49 68 19178
49 74 10282
83 81 9267
100 22 29727
46 81 31125
92 15 13741
62 58 25192
14 41 12920
44 4 27878
104 103 14251
33 90 17634
36 7 7019
40 61 11735
67 28 28525
2 51 14544
11 42 37
71 82 29662
98 98 15877
98 62 5132
23 24 5828
38 82 7194
28 44 8876
6 81 4454
54 96 28244
23 52 15809
83 15 9053
60 25 6194
51 69 29446
64 58 1993
102 91 31665
41 83 6931
44 22 32434
92 64 21461
1 62 11773
66 6 6665
103 46 12687
47 18 6754
30 98 21161
17 51 5049
16 89 3889
94 70 13361
49 57 16851
10 40 29179
62 16 7106
101 101 5366
46 63 29476
70 66 15785
7 62 30084
44 56 27458
102 32 12273
23 73 9637
49 43 9862
43 64 20910
51 19 29588
100 24 9515
35 103 10918
85 15 32072
71 22 5644
30 69 21849
19 75 26054
26 70 5442
36 20 7379
35 7 13819
58 36 1618
17 95 8152
40 52 19967
66 78 31131
21 33 4081
70 76 12418
95 59 24988
16 71 8392
78 102 2697
87 8 17242
54 43 27976
66 8 28838
11 33 4473
36 61 29135
62 21 26127
52 9 4278
13 20 16146
103 45 27787
22 5 21640
57 85 23526
3 43 12374
49 45 26273
12 92 5935
90 63 12507
96 89 7411
103 22 24058
5 67 20229
43 80 19838
91 5 16258
52 47 27904
10 77 21094
11 56 452
38 41 2555
64 32 4492
102 69 16937
92 43 17857
1 22 25212
1 96 5389
30 97 7421
12 54 32495
88 9 25658
8 2 506
73 63 27906
10 51 187
82 2 24775
78 52 21113
33 79 27559
84 89 5200
52 78 7557
96 32 17488
56 104 14275
72 64 5031
78 18 2410
56 6 32267
39 22 31221
1 92 25550
84 67 26357
21 2 29038
27 68 4927
11 56 23160
64 99 9840
45 91 10012
48 54 12769
50 22 23237
39 30 27751
81 84 24456
91 73 26739
7 75 13651
65 37 104
73 6 13596
27 62 9029
52 32 6021
93 9 8187
41 46 11176
25 60 6139
49 23 5649
52 60 29156
99 58 22797
24 101 2915
79 83 7887
93 20 5527
62 18 4905
54 59 2457
76 100 24000
65 24 2811
31 14 20228
76 99 27741
80 81 24924
56 71 28259
62 99 7484
58 98 19379
55 46 2593
37 54 17609
60 58 28589
23 70 13500
95 97 4440
69 2 21581
11 50 5221
15 80 17108
83 88 13097
62 78 17644
70 5 4550
102 48 30444
24 37 18401
13 71 1412
65 25 20476
29 17 26920
24 37 22657
49 93 12552
43 9 15265
11 38 12476
20 86 31371
79 60 28671
50 19 19886
63 76 7683
6 15 32520
53 83 4674
71 83 9381
11 81 8928
87 100 16605
84 36 23797
52 56 6908
98 60 1369
22 19 7759
31 69 18630
98 69 1740
45 31 14448
58 33 27091
1 69 11364
79 51 3769
83 50 30532
9 17 24090
39 2 3906
7 30 16420
103 16 32361
8 69 24924
10 10 8333
42 46 21135
71 63 32644
5 38 9188
88 80 3691
85 6 17487
19 43 11049
59 41 26075
3 32 14709
84 50 15480
11 76 27896
22 75 28761
58 101 117
24 85 24069
68 41 13813
95 79 3496
31 59 8291
13 41 30791
44 12 28968
104 85 3291
65 104 25757
7 92 16527
51 47 32373
43 94 7311
67 80 9062
29 15 286
45 94 20413
104 53 2303
69 48 17057
70 88 26956
7 38 1076
45 1 10574
25 74 16590
58 26 27384
46 94 7684
66 96 2063
89 50 32325
43 12 20783
14 85 11811
29 6 4889
40 87 17072
100 40 31775
17 35 4721
98 87 24201
95 46 15054
98 53 28221
87 51 8725
39 57 18199
1 18 27237
58 24 5812
54 64 25624
70 86 32669
65 64 20104
99 70 1296
18 15 11330
11 8 21988
42 98 11274
93 69 29228
63 53 14900
96 6 15024
15 25 17986
81 36 19033
60 74 14023
62 23 22799
45 15 31232
66 96 9423
10 22 11171
15 29 18653
102 29 30926
83 33 21591
78 2 20770
17 4 10395
87 2 8298
26 28 32057
75 96 20070
20 67 10865
92 68 10883
29 83 23651
68 63 18360
15 42 24704
67 52 12468
11 74 11227
1 57 22307
47 92 21612
87 76 16573
94 100 20583
56 39 19280
97 15 13865
71 99 32008
87 54 15481
79 47 18596
58 27 11781
62 4 1371
97 1 13408
95 76 10391
35 72 7670
4 10 9177
42 78 24492
19 38 13143
74 68 24101
61 96 31294
22 53 21473
41 42 3997
7 59 16570
24 72 11098
94 71 1120
80 21 1230
27 87 26857
83 65 15566
104 22 17297
20 36 27477
103 33 16700
53 78 11526
88 90 16725
22 7 26430
75 25 7458
68 84 31094
95 29 12970
7 103 18711
93 4 20920
73 3 4376
42 25 3743
23 12 12547
52 92 20950
1 49 28453
38 64 3195
104 3 20511
45 101 6492
22 58 23058
33 12 20264
1 47 25624
5 100 29630
30 11 717
36 97 27109
65 55 18700
31 58 27086
98 17 56
15 9 10795
88 40 15777
48 35 25288
9 45 25318
68 95 21215
13 36 13668
56 16 17571
46 74 23828
89 61 13497
36 50 21860
11 28 8085
80 49 15082
39 82 20467
46 6 30477
88 85 12519
1 43 9970
79 46 27446
5 44 19785
1 17 18682
42 72 30424
63 73 19393
16 24 2004
11 20 24730
76 88 31843
1 68 31450
76 33 4319
63 31 27009
74 103 5811
28 69 17529
45 71 16617
2 91 9774
86 85 32701
14 25 23346
60 18 9433
42 71 5352
49 62 4611
1 53 19811
32 101 12894
81 62 14860
95 15 663
13 102 14239
66 73 1675
46 77 21918
90 82 19281
102 26 24666
84 38 2733
33 4 20495
86 66 16718
14 95 10452
11 86 7023
14 58 27211
78 82 5037
86 19 32586
101 75 3633
38 63 32005
70 34 11909
8 91 15963
52 4 12452
57 20 21318
56 67 15913
48 8 9655
4 36 7849
15 13 27700
59 21 20127
11 95 22542
18 67 2667
9 35 12319
41 38 17245
39 41 2615
6 88 21179
83 3 25809
7 13 9029
77 19 20384
2 22 939
68 92 27925
75 71 29923
28 94 12277
75 85 7533
48 47 8072
9 49 9957
63 42 29203
32 28 15725
53 103 16543
39 30 16602
49 62 18479
25 100 1452
12 92 27640
3 82 22574
103 59 18753
10 24 24158
90 43 30949
81 80 1381
6 43 25526
8 76 28103
31 97 9782
10 32 21911
6 92 13654
3 51 25170
53 47 6470
65 92 29740
62 42 29811
94 44 27378
5 96 31816
47 93 2263
81 44 22099
27 87 31165
6 16 18386
26 64 1496
81 78 7902
98 28 11462
22 40 937
18 78 25197
42 87 17802
19 100 31034
77 56 2767
53 68 32152
73 27 23431
35 59 29885
79 5 7882
20 8 9496
30 52 7974
28 5 22790
98 4 22070
29 92 31905
55 29 26767
64 32 7609
65 61 12941
34 7 16733
31 16 23459
7 81 17540
95 35 26933
10 89 9640
86 17 2861
87 3 13180
7 43 11210
39 6 16646
9 18 31056
101 93 14777
10 35 31660
40 6 20584
92 25 8492
57 42 6703
34 22 21708
8 76 17055
46 15 30162
30 95 14542
70 94 15282
47 93 19744
94 98 618
82 79 17885
62 56 31442
32 18 1273
37 54 6050
59 2 915
42 79 6928
3 1 14355
99 15 12889
92 23 503
21 34 25489
52 88 682
77 82 1931
23 7 27897
94 79 27843
76 68 23817
86 62 24959
11 11 18640
89 53 4783
70 24 7561
68 77 14895
63 24 14486
59 61 21117
21 96 10955
30 55 16120
16 9 11041
4 104 24391
44 88 23092
101 58 10626
96 28 2595
57 87 31266
86 104 13993
104 2 6641
72 2 18308
42 95 22889
88 58 11147
59 72 24139
40 16 25854
104 28 11085
12 22 20469
66 81 21292
68 83 20986
83 8 32665
55 13 20020
41 91 18856
25 43 15274
10 84 9418
46 61 9095
87 48 7809
33 6 11111
12 14 17486
77 36 13335
44 68 28297
67 103 3916
30 60 18971
96 103 28473
24 42 5209
47 7 2323
61 72 25459
45 68 32738
85 15 23346
17 85 7983
61 68 8437
18 45 17770
54 52 4914
20 74 11305
2 49 2024
19 71 4229
50 2 29371
1 103 10303
78 83 7875
95 100 24702
59 82 28960
88 18 10554
92 95 16639
100 62 7349
43 75 1682
71 22 13455
60 80 17952
40 52 1626
49 103 32073
56 26 16313
55 7 18433
75 65 24170
15 64 16373
8 12 31445
45 1 5215
73 74 31722
23 103 20839
78 102 16766
68 21 5544
100 103 20067
15 14 22907
17 80 19946
43 94 16133
1 53 2205
26 88 29512
6 8 2629
70 38 18340
10 34 28575
24 77 10378
56 47 7245
4 92 16714
61 66 2533
7 18 17998
88 69 1838
63 40 22041
91 25 10979
67 57 30431
7 56 32671
43 45 14750
84 10 5818
92 13 19697
98 87 20029
10 86 6430
16 39 14905
86 40 11276
6 61 23590
31 20 5921
9 25 13078
46 33 14906
61 52 17881
89 7 7742
32 97 9325
75 9 18163
64 95 4173
89 51 5928
30 15 4985
66 29 21320
22 51 23631
47 54 19960
5 50 18576
38 17 10062
91 8 13344
9 39 4796
69 16 28772
7 66 8397
36 36 20303
84 25 24896
99 24 31096
9 85 10025
59 72 26835
8 97 7196
103 62 27103
63 31 28816
30 23 875
102 76 9982
30 22 10556
9 69 1125
38 19 14394
52 61 27704
40 52 31478
52 78 9963
76 26 27549
44 94 6540
58 75 9402
72 29 6708
38 40 1103
81 77 8246
71 3 32053
47 67 24652
100 13 192
27 81 17466
3 39 24338
3 53 27297
84 73 1713
77 61 18160
48 55 15410
96 58 25270
37 31 18708
12 84 30924
45 91 19754
68 8 30728
34 80 24610
9 70 15986
36 16 6402
84 56 24383
60 64 28605
82 26 21753
91 95 20779
96 47 28312
78 14 14343
77 13 24958
13 12 15808
48 54 913
44 87 27714
30 98 6123
34 91 13824
88 45 7638
92 83 27625
50 11 1289
29 65 24843
1 19 23403
8 20 14864
53 95 18372
60 42 15068
57 42 16923
98 76 20216
88 4 17853
78 102 6535
29 81 544
41 69 8182
94 39 10005
32 88 11955
29 90 7698
35 94 31109
32 42 9591
41 57 16729
97 62 9236
12 28 1960
86 31 31334
47 57 7102
90 77 25661
71 93 10373
17 10 30921
1 48 27053
83 18 20548
35 101 25919
22 51 7604
71 12 761
1 7 28784
56 18 11539
45 68 9335
103 10 24114
15 85 6590
42 2 11958
50 32 27116
2 17 10150
14 31 20818
77 103 27628
51 62 11004
22 85 14491
53 64 2339
1 56 25452
18 100 27250
5 97 27338
69 98 8673
40 19 27891
76 31 2565
62 32 1054
103 57 24106
71 25 9127
19 90 25247
91 37 13997
89 63 13037
73 66 31405
88 6 26610
84 48 28918
53 2 8477
18 2 22541
53 41 9021
42 27 31324
4 99 1178
59 27 20932
17 74 4442
6 29 6114
99 24 1115
58 69 26922
20 44 12299
10 63 13608
4 85 4064
61 83 24710
5 57 18053
49 100 19878
39 38 5591
38 94 1775
81 74 13629
68 21 4647
81 17 24588
8 71 5733
78 84 22770
76 53 1671
54 34 16534
19 80 47
88 85 19706
95 90 5040
35 31 31807
93 91 17271
35 87 6353
29 5 18898
14 93 7731
34 63 10787
41 77 5342
2 58 20193
81 2 8428
102 96 2037
70 84 4744
73 75 13651
27 45 6236
104 29 22501
34 52 4984
23 73 14107
33 83 18780
57 95 21438
6 55 7216
54 4 4320
100 52 23125
94 80 25079
10 25 28108
84 93 31106
28 22 2348
99 33 30163
77 73 13079
59 25 29134
37 33 4722
9 96 15803
21 71 27856
58 64 26751
17 98 13771
33 65 15322
25 87 10557
28 62 28280
62 36 926
41 20 9466
68 13 21233
32 40 30528
103 12 24575
87 79 8605
101 2 8922
9 83 25022
24 28 26329
92 100 19807
20 54 397
31 89 17717
21 2 7519
75 61 4235
31 38 25556
22 36 16001
100 42 7348
35 6 12283
37 24 28280
85 40 14739
5 34 8664
33 35 12084
41 22 27979
90 11 29482
102 69 11989
1 83 24769
51 104 27001
69 63 11986
90 11 10538
7 44 16591
42 60 27408
98 43 23842
20 30 8567
