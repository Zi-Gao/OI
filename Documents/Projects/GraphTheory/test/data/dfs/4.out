1 7 13 12 14 25 43 9 17 4 10 22 5 34 52 32 18 2 49 23 24 28 44 56 6 8 20 30 11 33 35 31 16 39 38 19 71 3 51 47 54 59 21 96 58 26 64 95 15 29 65 37 55 46 61 66 73 27 45 68 41 42 60 74 103 57 85 40 87 48 76 53 78 82 79 83 50 81 36 97 62 99 70 84 67 80 89 63 93 69 98 91 86 104 88 90 77 94 100 102 72 92 101 75 